docs/user-guide/glossary/edit-glossaries.md

```diff
@@ -1,6 +1,28 @@
 ### Prerequisite
 See [glossary roles and permission](/user-guide/glossary/glossary-roles-permissions) for permission setup.
 
+### From Glossary page
+1. Click `Glossary` menu.
+<figure>
+![Glossary page](/images/glossary-page.png)
+</figure>
+1. Go to entry you wish to edit, fill in part of speech or description.
+<figure>
+![Glossary edit](/images/glossary-edit.png)
+</figure>
+1. Click `Update` to save your changes, or `Cancel` to revert your changes, `Delete` to delete this entry.
+1. To edit translation, select language from list.
+<figure>
+![Glossary edit](/images/glossary-select-language.png)
+</figure>
+1. Fill in the translation, and click `Update`.
+<figure>
+![Glossary edit](/images/glossary-edit-trans.png)
+</figure>
+
+
+### From editor
+
 1. Go to editor in Zanata and select a string. Instructions for opening the
     translation editor can be found at [Preparing for Translation](/user-guide/translator-guide#start-translate-a-project-version).
 1. In Glossary panel (bottom right corner), click on <i class='i i--info txt--highlight'></i> icon on the glossary entry you wish to edit.
```

Comment:
The point below here changes from telling the user directly what to do to talking about users in the third person, should make it consistent.